README.md

```diff
@@ -6,41 +6,114 @@
 
-[Amazon SNS](https://aws.amazon.com/sns/) is a message notification service developed by Amazon.
+[Amazon SNS](https://aws.amazon.com/sns/) is a message notification service provided by Amazon.com Inc., enabling users to publish messages to topics, which are then delivered to subscribing endpoints or clients.
 
-This connector provides operations for connecting and interacting with Amazon SNS endpoints over the network. Its main capabilities are to define SNS topics, publish messages, and to subscribe email, sms and sqs etc. to SNS topics. Following modules are available in the connector. 
+The `ballerinax/aws.sns` package offers APIs to connect and interact with [AWS SNS API](https://docs.aws.amazon.com/sns/latest/api/welcome.html) endpoints.
 
-- [`aws.sns`](sns/Module.md)
+## Quickstart
 
-## Building from the source
-### Setting up the prerequisites
+**Note**: Ensure you follow the [prerequisites](https://github.com/ballerina-platform/module-ballerinax-aws.sns#set-up-aws-sns-api) to set up the AWS SNS API.
 
-1. Download and install Java SE Development Kit (JDK) version 11. You can install either [OpenJDK](https://adoptopenjdk.net/) or [Oracle JDK](https://www.oracle.com/java/technologies/javase-jdk11-downloads.html).
+To use the `aws.sns` connector in your Ballerina application, modify the `.bal` file as follows:
 
-    > **Note:** Set the JAVA_HOME environment variable to the path name of the directory in which you installed JDK.
+### Step 1: Import the connector
+Import the `ballerinax/aws.sns` package into your Ballerina project.
+```ballerina
+import ballerinax/aws.sns;
+```
 
-2. Download and install [Ballerina Swan Lake](https://ballerina.io/). 
+### Step 2: Instantiate a new connector
+Create a `sns:ConnectionConfig` record with the obtained `accessKeyId` and `secretAccessKey` and initialize the connector with it.
+```ballerina
+sns:ConnectionConfig config = {
+    accessKeyId: accessKeyId,
+    secretAccessKey: secretAccessKey,
+    region: region
+};
 
-### Building the source
-Execute the following commands to build from the source:
+sns:Client amazonSNSClient = check new(config);
+```
 
-* To build the package:
-    ```    
-    bal build ./ballerina
-    ```
-* To run tests after build:
+### Step 3: Invoke the connector operation
+Now, utilize the available connector operations.
+```ballerina
+string topicArn = check amazonSNSClient->createTopic("FirstTopic");
+```
+
+For comprehensive information about the connector's functionality, configuration, and usage in Ballerina programs, refer to the `aws.sns` connector's reference guide in [Ballerina Central](https://central.ballerina.io/ballerinax/aws.sns/latest).
+
+## Set up AWS SNS API
+
+1. Create an [AWS account](https://portal.aws.amazon.com/billing/signup)
+
+2. [Obtain access keys](https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html)
```

Comment:
Shall we have screenshots for setting up? and move this to up.

Reply:
I've added a separate [setup guide](https://github.com/kaneeldias/module-ballerinax-aws.sns/blob/docs/docs/setup/setup.md) similar to the gmail connector.